src/openrct2/Game.h

```diff
@@ -18,6 +18,124 @@
 class Intent;
 struct ParkLoadResult;
 
+// TODO: Move or remove these constants if
+//       we can use the constants defined
+//       in Context.h
+namespace TickConversion
+{
+    constexpr uint32_t kGameUpdateFPS = 40;
+    constexpr float kGameUpdateFPMS =  kGameUpdateFPS / 1000.0f;
```

Comment:
We don't need floats, the value should be milliseconds as integer, so this should result in 25, 1000 / 40 = 25
```suggestion
    constexpr float kGameUpdateFPMS =  kGameUpdateFPS / 1000.0f;
```